### PR TITLE
i18n(fr): Translation of `buddy.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `buddy.mdx` to french

### CODE REVIEW

File: src/pages/fr/guides/deploy/buddy.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Vous pouvez déployer votre projet Astro en utilisant [Buddy](https://buddy.works/), une solution CI/CD qui peut construire votre site et le pousser vers de nombreuses cibles de déploiement différentes, y compris les serveurs FTP et les fournisseurs d'hébergement en nuage.
+
+:::note
+Buddy ne hébergera pas votre site lui-même. Au lieu de cela, il vous aide à gérer le processus de compilation et à livrer le résultat à une plate-forme de déploiement de votre choix.
+:::
+
+## Comment déployer
+
+1. [Creér un compte](https://buddy.works/sign-up).
```
Comment (thomasbnt) [inline src/pages/fr/guides/deploy/buddy.mdx:16]: "Créez un compte"

File: src/pages/fr/guides/deploy/buddy.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Déployez votre site Astro avec Buddy
+description: Comment déployer votre site Astro sur le web en utilisant Buddy.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+Vous pouvez déployer votre projet Astro en utilisant [Buddy](https://buddy.works/), une solution CI/CD qui peut construire votre site et le pousser vers de nombreuses cibles de déploiement différentes, y compris les serveurs FTP et les fournisseurs d'hébergement en nuage.
```
Comment (thomasbnt) [inline src/pages/fr/guides/deploy/buddy.mdx:8]: `hébergement en nuage.`

Seems not a correct translation, "cloud" is not better like `hébergement sur un cloud` ?

**Review by thomasbnt — CHANGES_REQUESTED** [review]

**Comment by dreyfus92** [comment]
Merci pour le review @thomasbnt 😄

**Comment by Yan-Thomas** [comment]
Thanks @thomasbnt and @dreyfus92, great work!

**Pull request merged**